app/controllers/api/account.php

```diff
@@ -2353,14 +2352,17 @@
                 // Use current session for events.
                 $queueForEvents->setPayload($response->output($session, Response::MODEL_SESSION));
             }
+
+            $queueForEvents
+                ->setEvent('users.[userId].sessions.[sessionId].delete')
+                ->setParam('userId', $user->getId())
+                ->setParam('sessionId', $session->getId())
+                ->setPayload($response->output($session, Response::MODEL_SESSION))
+                ->trigger();
```

Comment:
will the last event trigger twice since our shutdown hook will also trigger an event?